### PR TITLE
plat-stm32mp1: fix helper function and increase CFG_DTB_MAX_SIZE

### PR DESCRIPTION
**Pull request by etienne-lms**

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: core/arch/arm/plat-stm32mp1/conf.mk [hunk 1]
```diff
@@ -60,6 +60,7 @@ CFG_TEE_CORE_NB_CORE ?= 2
 CFG_WITH_PAGER ?= y
 CFG_WITH_LPAE ?= y
 CFG_MMAP_REGIONS ?= 23
+CFG_DTB_MAX_SIZE ?= 0x40000
```
Comment (jforissier) [inline core/arch/arm/plat-stm32mp1/conf.mk:63]: I personally prefer `262144` or even better `(256 * 1024)` should work -- we have this already:
```
core/arch/arm/arm.mk:CFG_RESERVED_VASPACE_SIZE ?= (1024 * 1024 * 10)
```

**Comment by jforissier** [comment]
Regardless of how you prefer to write the 256 KB value...

`Acked-by: Jerome Forissier <jerome@forissier.org>`

**Comment by etienne-lms** [comment]
tags applied.

**Pull request merged**